com.unity.ml-agents/Documentation~/index.md

```diff
@@ -1 +1,71 @@
-Please see the [ML-Agents README)(https://github.com/Unity-Technologies/ml-agents/blob/master/README.md)
\ No newline at end of file
+# About Unity MLAgents package
```

Comment:
according to the [Package Starter Kit](https://github.cds.internal.unity3d.com/unity/com.unity.package-starter-kit#package-structure) I think this file is supposed to be named `com.unity.ml-agents.md`.